src/dpl/include/dpl/Opendp.h

```diff
@@ -617,7 +617,7 @@ class Opendp
   std::unique_ptr<DplObserver> debug_observer_;
 
   // Magic numbers
-  static constexpr int bin_search_width_ = 10;
+  static constexpr int bin_search_width_ = 2;
```

Comment:
Why 2 ?

Reply:
this was the bug that made the PQ approach not work as expected. The way it was used is that after a bin is identified from the old diamond search, there was a for loop that checked 10 bins to the left and right of it, and if any of them were available, it returned them.

there could have been an empty bin to the top at a shorter Manhattan distance that would not be discovered because the 9th bin on the row below it was available.

2 or 1 should work, but I experimented with all numbers from 1 to 10, and 2 was slightly better than 1 always (and a tiny bit faster).